packages/jest/README.md

```diff
@@ -66,6 +66,7 @@ The builder supports multi-project workspaces out of the box, the only thing req
 
    Thus, if you don't provide `configPath` in options, and you'd like to customize the configuration of a single project in your workspace, you only have to add _jest.config.js_ in this project's root directory and specify the configuration delta in this file.  
    Or, if you'd like the same custom configuration to be applied to all the projects in the workspace, you just specify it in _package.json_. Another option in such a case is creating a single config file in the workspace root and specifying it in _angular.json_ for each project.
+ - `tsConfig` - path to tsconfig file. Defaults to `tsconfig.spec.json` that is located in _project root_.
```

Comment:
Maybe worth to specify that if the path is relative then it is evaluated relative to the project root.